Sources/MapboxDirections/RouteResponse.swift

```diff
@@ -128,13 +130,15 @@ extension RouteResponse: Codable {
                 let waypoint = Waypoint(coordinate: decodedWaypoint.coordinate,
                                         coordinateAccuracy: waypointInOptions.coordinateAccuracy,
                                         name: waypointInOptions.name?.nonEmptyString ?? decodedWaypoint.name)
-
+                waypoint.snappedDistance = decodedWaypoint.snappedDistance
```

Comment:
For some reason not all of the Waypoint properties were copied here. Was that intended?

Reply:
Probably an oversight.

Reply:
The changelog should mention that we’ve fixed the issue where `Waypoint.snappedDistance` was getting lost when decoding a RouteResponse.